rtslib/root.py

```diff
@@ -184,9 +184,12 @@ def _get_saveconf(self, so_path, save_file):
         '''
         current = self.dump()
 
-        with open(save_file, "r") as f:
-            saveconf = json.loads(f.read())
-            f.close()
+        if not os.path.exists(save_file):
```

Comment:
Instead of testing, wrap, this in a try/except block?

BTW it also doesn't need to close f (it gets closed when execution leaves the `with` scope) so while you're in there you can zap that.

Reply:
Sure @agrover, I will fix it. Thanks!